cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -366,13 +374,30 @@ public Config setCreateBusbarSectionForEveryConnectivityNode(boolean b) {
             return this;
         }
 
+        public FlowProfile getProfileUsedForInitialFlowsValues() {
+            return profileUsedForInitialFlowsValues;
+        }
+
+        public Config setProfileUsedForInitialFlowsValues(String profileUsedForInitialFlowsValues) {
```

Comment:
I prefer `setProfileUsedForStateHypothesis`

Reply:
Same comment for the rest of the code. Before making any change, maybe wait for Luma's name suggestion.